app/views/notices/shop_employee_notices/13_employee_open_enrollment_reminder.html.erb

```diff
@@ -39,22 +35,29 @@
   { content: "Open enrollment is your annual opportunity to enroll in your employer’s health plan or make changes to your current plan selection.  Outside of this annual open enrollment period, you will only be allowed to make changes to your health plan enrollment if you experience certain life events, such as marriage, birth, or adoption, or other qualifying events."} %>
 
 <%= render partial: 'notices/shared/paragraph_with_headers', :locals =>
-  { contents: [""], heading: 'For Questions or Assistance:', sub_heading: ''} %>
-
-
+    { heading: 'For Questions or Assistance:',
+      col_styles: ['margin-bottom: 0px']} %>
+<%= render partial: 'notices/shared/paragraph', :locals =>
+  { content: "Please contact your broker for further assistance. You can also contact the #{site_short_name} with any questions:"} %>
 
-<%= render partial: 'notices/shared/unordered_list', :locals => { listings:
-                                                                   ["By calling <a href='#{contact_center_phone_number}'>#{contact_center_phone_number}</a>",
-                                                                     "TTY: #{contact_center_tty_number}",
-                                                                     "Online at #{site_main_web_link}"] } %>
+<table class="total_table" border="0">
+  <tbody>
+    <tr>
+      <td>
+        <ul style="list-style: circle solid;">
+        <li class="notice-font">
+            By calling <%=contact_center_phone_number%>   &nbsp; TTY: <%=contact_center_tty_number%><br>
+        </li>
+        <li class="notice-font">
+            By email: <a mail_to= "#{small_businesss_email}"> <%=small_businesss_email%></a> 
```

Comment:
Try defining the link this way.
 <%= mail_to "#{small_businesss_email}", "#{small_businesss_email}", target: "_blank"%>